libc/src/__support/GPU/amdgpu/utils.h

```diff
@@ -176,6 +176,12 @@ LIBC_INLINE uint64_t fixed_frequency_clock() {
     return 0;
 }
 
+/// Terminates execution of the associated wavefront.
+LIBC_INLINE void exit() {
+  __builtin_amdgcn_endpgm();
+  __builtin_unreachable();
```

Comment:
This should be redundant, the endpgm should be noreturn

Reply:
So it can be omitted for AMDGPU, thanks.